pages/common/pwsh.md

```diff
@@ -0,0 +1,20 @@
+# pwsh
+
+> PowerShell Core is a cross-platform automation and configuration tool/framework.
+> More information: <https://docs.microsoft.com/en-us/powershell/>.
+
+- Start an instance of PowerShell:
+
+`pwsh`
+
+- Run the specified script:
+
+`pwsh -File {{path/to/file.ps1}}`
+
+- Set the execution policy for the current session:
```

Comment:
Is there maybe an easier description for `execution policy?`

Reply:
And with `current` do you mean the session you're executing this command in or the session which will be started by this command?

Reply:
That's a very good question. 🤔 

```
-ExecutionPolicy
    Sets the default execution policy for the current session and saves it in the $env:PSExecutionPolicyPreference environment variable. This  pa‐
    rameter does not change the PowerShell execution policy that is set in the registry.
```

I'm testing it a bit, but whenever I do `Get-ExecutionPolicy` is always returns `Unrestricted`. 🤔

Reply:
Well, at least it's not worse than the official docs

Reply:
It's probably best to stick with  the official language here, even if it doesn't make a whole lot of sense I guess?